doc/distrib/NodeHelpFiles/_review/DSRevitNodesUI.DetailLevel.md

```diff
@@ -0,0 +1,14 @@
+## In Depth
+`Detail Level` allows for the selection of a view detail level for use in graphic settings.
+
+Available detail levels available include:
```

Comment:
Available in this sentence is repetitive. Maybe rephrase as "Detail levels available include:"